static/app/utils/profiling/profile/chromeTraceProfile.tsx

```diff
@@ -510,24 +514,151 @@ export function collapseSamples(profile: ChromeTrace.CpuProfile): {
   return {samples, sampleTimes};
 }
 
-export function parseChromeTraceFormat(
+function shouldPlaceOnTopOfPreviousStack(_frame: ChromeTrace.CallFrame): boolean {
+  return false;
+}
+
+function lastOf<T extends any[]>(arr: T): T[number] {
+  return arr[arr.length - 1];
+}
+
+const callFrameToFrameInfo = new Map<ChromeTrace.CallFrame, Frame>();
```

Comment:
Where do you `set` values inside of this map? I only see a `get` just below here but no `set` anywhere.